src/serverenvironment.cpp

```diff
@@ -413,6 +413,18 @@ ServerEnvironment::ServerEnvironment(ServerMap *map,
 	std::string name;
 	conf.getNoEx("player_backend", name);
 	m_player_database = openPlayerDatabase(name, path_world, conf);
+
+	std::string auth_name = "files";
+	if (conf.exists("auth_backend")) {
+		conf.getNoEx("auth_backend", auth_name);
```

Comment:
does this even work without a `&`?

Reply:
The `getNoEx` method is defined as
```bool getNoEx(const std::string &name, std::string &val) const;
```
The second parameter is pass-by-reference, and not const, so it can be assigned to.

It also works for the player backend a few lines above, which already existed in code.